_extras/etherpad.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: base
+title: "Etherpad Template"
+---
+<blockquote>
+<p>Welcome to The Carpentries Etherpad!</p>
+
+<p>This pad is synchronized as you type, so that everyone viewing this page sees the same text. This allows you to collaborate seamlessly on documents.</p>
+
+<p>Use of this service is restricted to members of The Carpentries community; this is not for general purpose use (for that, try etherpad.wikimedia.org).</p>
+
+<p>Users are expected to follow our code of conduct: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html</p>
+
+<p>All content is publicly available under the Creative Commons Attribution License: https://creativecommons.org/licenses/by/4.0/</p>
+<br/> ____________________________________________________________________________________________________________
+<br/>
+<b>Sign in: Name, Institution, Email, Twitter (optional)</b><br/>
+Please sign in so we can record your attendance.<br/>
+<ul>
+  <li>&nbsp;</li>
+  <li>&nbsp;</li>
+  <li>&nbsp;</li>
+  <li>&nbsp;</li>
+</ul>
+<br/>
+Please fill out the pre-training survey at https://www.surveymonkey.com/r/instructor_training_pre_survey?workshop_id=2019-07-18-ttt-macquarie
+ 
+</blockquote> 
+
+
+
+
+{% comment %}
+Create anchor for each one of the episodes.
+{% endcomment %}
+
+{% for episode in site.episodes %}
+<h1>Episode: {{episode.title}} <a href="{{site.url}}{{site.baseurl}}{{episode.url}}">{{site.url}}{{site.baseurl}}{{episode.url}}</a></h1>
```

Comment:
This would not render locally. Consider using `{{ relative_root_path }}`. Even better -- have a look at `_includes/all_keypoints.html`

Reply:
Question: I don't see relative_root_path in all keypoints, though this suggestion is really good because it reminded me to add the markdownify filter.

Looking at this, I *think* I'm getting what you're saying. You're suggesting that baseurl isn't operating the way we need it to do and we should swap baseurl for relative-root-path, rather than site.url *and* baseurl. Since I've started editing this, I'm going to throw a commit up for your review.

Reply:
> I don't see relative_root_path in all keypoints

This means that this lesson hasn't been synced up with https://github.com/carpentries/styles in a while. `relative_root_path` is defined in here: https://github.com/carpentries/styles/blob/gh-pages/_includes/base_path.html  and all_keypoints are here: https://github.com/carpentries/styles/blob/gh-pages/_includes/all_keypoints.html

> You're suggesting that baseurl isn't operating the way we need it to do and we should swap baseurl for relative-root-path, rather than site.url and baseurl.

If I `jekyll build` the site, the links will be `/01-episode-name/index.html` and so on. With `relative_root_path` they will be correct

Reply:
Actually, there is `all_keypoinds.html` in this repo, see https://github.com/carpentries/instructor-training/blob/gh-pages/_includes/all_keypoints.html

Reply:
@maxim-belkin Yes, the last time I pulled my repo from this one was before the June push. Thank you for the updated reference. I've changed {{site.baseurl}} to be {{relative_root_path}} as per your request. However, we still require {{site.url}} otherwise the pad.carpentry.org links will point to other etherpads instead of lessons. My links are absolute links such that these notes resolve to the original lesson regardless of where they are pasted. Sorry, I feel like I'm not communicating well this morning.

Reply:
> I've changed {{site.baseurl}} to be {{relative_root_path}} as per your request. However, we still require {{site.url}} otherwise the pad.carpentry.org links will point to other etherpads instead of lessons.

I think you're right. I was thinking in terms of links within a website.

Reply:
All good. The ghost of wittgeinstein is laughing at our language games. :) In terms of relative_root_path v. baseurl, do you have a sense is which is more appropriate to use with site.url?